.markdownlint.jsonc

```diff
@@ -1,10 +1,17 @@
 {
   "default": true,
+  "MD003": { "style": "atx" },
```

Comment:
can we also add a ref to the rule as is done with the others? I get most of these but it would still be good to keep a reference to them handily around.